dape.el

```diff
@@ -1186,12 +1098,12 @@ See `dape--callback' for expected CB signature."
                    :lines (apply 'vector lines))
                   cb)))
 
-(defun dape--set-exception-breakpoints (process cb)
-  "Set the exception breakpoints in adapter PROCESS.
+(defun dape--set-exception-breakpoints (conn cb)
```

Comment:
These functions which take `cb` callback-passing style are IMO harder to read than imperative counterparts.  I think you get the point from my other PR.